test/common/ssl/test_data/not_a_crl.crl

```diff
@@ -0,0 +1,3 @@
+-----BEGIN X509 CRL-----
```

Comment:
Could you replace this with inline test?

Reply:
I don't think I can do that easily since the v1 config doesn't support inline data, AFAIK.  Happy to write out a temporary file during the test, perhaps?

Reply:
It's fine to replace the existing v1 test with v2 test, since that's the recommended syntax anyway... 

Also, there should be v2 tests for success / invalid content (i.e. replacement for this file) / invalid file path.

Reply:
Added success / failure cases to the v2 tests - do you want me to just remove the v1 tests in `test/common/ssl/context_impl_test.cc`?

Reply:
Up to you, I'm fine either way.